sdk/main/src/ED25519Keypair.cc

```diff
@@ -41,4 +43,38 @@ std::shared_ptr<PublicKey> ED25519Keypair::getPublicKey() const
 {
   return publicKey;
 }
+
+std::vector<unsigned char> ED25519Keypair::sign(std::vector<unsigned char> bytesToSign) const
```

Comment:
It doesn't look like this input vector is being edited in anyway, is there anyway to make it a const reference? Not sure what the openssl functions want as inputs but worth checking